execution/types/runtime_decoder_test.go

```diff
@@ -0,0 +1,343 @@
+package types
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+	sc "github.com/LimeChain/goscale"
+	"github.com/LimeChain/gosemble/mocks"
+	primitives "github.com/LimeChain/gosemble/primitives/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+var (
+	mockModuleOne        *mocks.Module
+	mockCallOne          *mocks.Call
+	moduleOneIdx         = sc.U8(0)
+	moduleIdx            = sc.U8(0)
+	functionIdx          = sc.U8(0)
```

Comment:
not super important, but we usually tend to separate declarations of constants, variables, ...

```
const (
	moduleOneIdx = sc.U8(0)
	moduleIdx    = sc.U8(0)
	functionIdx = sc.U8(0)
)


var (
	signedExtrinsicBytes = []byte{
   ...
)

var (
	mockModuleOne *mocks.Module
	mockCallOne   *mocks.Call
)